source/examples/atlas-examples/AtlasSearchHelpers.kt

```diff
@@ -0,0 +1,54 @@
+import com.mongodb.client.model.Aggregates
+import com.mongodb.client.model.Projections
+import com.mongodb.client.model.search.SearchOperator
+import com.mongodb.client.model.search.SearchPath.fieldPath
+import com.mongodb.kotlin.client.coroutine.MongoClient
+import kotlinx.coroutines.runBlocking
+import org.bson.Document
+import org.bson.conversions.Bson
+import java.io.IO.println
+import kotlin.collections.List
+
+const val URI = "<connection-string>"
+
+// Create data class to represent a MongoDB document
+data class Movie(val title: String, val year: Int, val cast: List<String>)
+
+fun main() {
+
+    // Replace the placeholder with your MongoDB deployment's connection string
+    val uri = URI
+
+    val mongoClient = MongoClient.create(uri)
+    val database = mongoClient.getDatabase("sample_mflix")
+    // Get a collection of documents of type Movie
+    val collection = database.getCollection<Movie>("movies")
+
+    // start atlasHelperMethods
+    runBlocking {
+        val searchStage: Bson = Aggregates.search(
+            SearchOperator.compound()
+                .filter(
+                    listOf(
+                        SearchOperator.text(fieldPath("genres"), "Drama"),
```

Comment:
given `genres` is an array field, should we use `in` operator instead?

Reply:
Hi Nathan, I'll update this example to use the `in` operator here and perform a different `phrase` query (your other comment).